tests/saitests/py3/sai_qos_tests.py

```diff
@@ -183,6 +183,21 @@ def get_rx_port_pkt(dp, src_port_id, pkt, exp_pkt):
 
         return result.port
 
+
+    def is_rx_port(dp, src_port_id, dst_port_id, pkt):
+        asic_type = dp.test_params['sonic_asic_type']
+        queue = dp.test_params['pg']
+        _, base_queue_counters = sai_thrift_read_port_counters(dp.dst_client, asic_type, port_list['dst'][dst_port_id])
+        send_packet(dp, src_port_id, pkt, 10)
+        time.sleep(8)
+        _, queue_counters = sai_thrift_read_port_counters(dp.dst_client, asic_type, port_list['dst'][dst_port_id])
+        if queue_counters[queue] > base_queue_counters[queue]:
```

Comment:
why we have this API ? Also this ` if queue_counters[queue] > base_queue_counters[queue]:` check should see if we see increment of 10 pkts

Reply:
It's to avoid ptf not receiving jumbo packet issue. I have debugged the setup and ping with jumbo packets ok, but still ptf not receiving jumbo packets. So add this API to check which port in port-channel is the egress port.

Updated the if condition to check increment of 10 pkts.